### PR TITLE
Fix mobile notifications

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
We noticed some issues for mobile notifications

1. Error 

> Notification subtitle must not be null for any languages.

iOS require subtitle, then we add not blank validation

2. Process error in timeline

At the moment error from response is not set to database and not display in timeline. This PR added standard error message to timeline, for example: 

image

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. You should use mobile notification, what is really hard to setup
3. Then code review is good and basic test for mobile notification (create/edit)

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10062"></a>

### CODE REVIEW

**Review by mabumusa1 — APPROVED** [review]
The code is good to go after rebase

**Comment by kuzmany** [comment]
@mabumusa1 do not look like it need rebase

File: app/bundles/NotificationBundle/Views/SubscribedEvents/Timeline/index.html.php [hunk 1]
```diff
@@ -9,10 +9,11 @@
  * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
  */
 $data = $event['extra']['log']['metadata'];
-if (isset($data['failed'])) {
-    return;
-}
 ?>
+<?php if (isset($data['failed'])): ?>
+    <p class="text-danger mt-0 mb-10"><i class="fa fa-warning"></i> <?php echo $view['translator']->trans('mautic.campaign.event.last_error').': '.$data['reason']; ?></p>
+    <?php return; ?>
```
Comment (escopecz) [inline app/bundles/NotificationBundle/Views/SubscribedEvents/Timeline/index.html.php:15]: Wow, haven't seen this in a HTML view yet. Does it mean you don't want to show the rest of the view? Can this be replaced with an `else` statement instead? I cannot find how to do this in a Twig template. And we'll need to rewrite this PHP template to TWIG pretty soon.

**Review by escopecz — CHANGES_REQUESTED** [review]

**Comment by mabumusa1** [comment]
> @mabumusa1 do not look like it need rebase

@kuzmany I did that yesterday.

**Comment by kuzmany** [comment]
Close at the moment. We will push it again once ready to M5

**Pull request closed without merging**